### DISCUSSION TITLE
IFC format conversion

### DISCUSSION THREAD

**Discussion by someparsa** [mozman/ezdxf#821] [General]
Is there any feature or function with which we can export an IFC format of the DXF model we have coded?

**Comment by mozman** [comment]
You can export single MESH or POLYLINE based meshes by the `meshex`  add-on. The add-on does no extensive validation and there is no guarantee that every CAD application can open these IFC files.

Links:
- [documentation](https://ezdxf.mozman.at/docs/addons/meshex.html) of the `meshex` add-on
- example script [meshex_export.py](https://github.com/mozman/ezdxf/blob/master/examples/addons/meshex_export.py)

**Reply by someparsa** [reply]
Thank you, that may help. Will investigate.
